### PR TITLE
Migrate pages to Sandbox

### PR DESCRIPTION
**Pull request by sans-harness**
- Migrates most pages to sandbox except `pipeline-edit flow` and `execution-details`.

### CODE REVIEW

File: apps/gitness/src/pages/execution/sanbox-execution-details.tsx
Comment (joetaylor-harness) [inline apps/gitness/src/pages/execution/sanbox-execution-details.tsx:1]: typo in name.

File: apps/gitness/src/layouts/PullRequestSandboxLayout.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  }, [pullRequestData, isFetching])
+
+  const baseClasses =
+    'inline-flex items-center justify-center px-3 py-1 px-4 items-center gap-2 bg-background hover:text-primary h-[36px] rounded-tl-md rounded-tr-md m-0'
+  const getLinkClasses = (isActive: boolean) => {
+    return `${baseClasses} ${isActive ? 'text-primary [&svg]:text-primary tabnav-active' : 'tabnav-inactive'}`
+  }
+  return (
+    <>
+      <SandboxLayout.Main hasHeader hasLeftPanel>
```
Comment (joetaylor-harness) [inline apps/gitness/src/layouts/PullRequestSandboxLayout.tsx:32]: Are you planning to remove reference to 'Sandbox' in the future, and simply call it Layout?

File: apps/gitness/src/pages/pull-request/sandbox-pull-request-conversation-page.tsx
Comment (joetaylor-harness) [inline apps/gitness/src/pages/pull-request/sandbox-pull-request-conversation-page.tsx:1]: This is a massive file, kinda hard to read. Maybe in the future split it up?

File: apps/gitness/src/pages/pull-request/sandbox-pull-request-conversation-page.tsx [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+    codeOwners?.evaluation_entries,
+    CodeOwnerReqDecision.CHANGEREQ
+  )
+  const codeOwnerPendingEntries = findWaitingDecisions(codeOwners?.evaluation_entries)
+  const codeOwnerApprovalEntries = findChangeReqDecisions(codeOwners?.evaluation_entries, CodeOwnerReqDecision.APPROVED)
+  const latestCodeOwnerApprovalArr = codeOwnerApprovalEntries
+    ?.map(entry => {
+      // Filter the owner_evaluations for 'changereq' decisions
+      const entryEvaluation = entry?.owner_evaluations.filter(
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (joetaylor-harness) [inline apps/gitness/src/pages/pull-request/sandbox-pull-request-conversation-page.tsx:103]: Definitely can't improve on 'any' for typing?

File: apps/gitness/src/pages/pull-request/sandbox-pull-request-conversation-page.tsx [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
+  })
+  const { data: { body: activityData } = {} } = useListPullReqActivitiesQuery({
+    repo_ref: repoRef,
+    pullreq_number: prId,
+    queryParams: {}
+  })
+  const [changesLoading, setChangesLoading] = useState(true)
+
+  const [activities, setActivities] = useState(activityData)
+  const approvedEvaluations = reviewers?.filter(evaluation => evaluation.review_decision === 'approved')
```
Comment (joetaylor-harness) [inline apps/gitness/src/pages/pull-request/sandbox-pull-request-conversation-page.tsx:82]: Could filters be memoized?

**Review by joetaylor-harness — APPROVED** [review]

File: apps/gitness/src/layouts/PullRequestSandboxLayout.tsx (hunk 1 above)
Comment (abhinavrastogi-harness) [reply apps/gitness/src/layouts/PullRequestSandboxLayout.tsx:32]: +1

File: apps/gitness/src/pages/execution/sanbox-execution-details.tsx
Comment (abhinavrastogi-harness) [reply apps/gitness/src/pages/execution/sanbox-execution-details.tsx:1]: we should remove `sandbox` from filenames as well.. once we remove the other versions (in a separate PR maybe)

File: apps/gitness/src/pages/pull-request/sandbox-pull-request-conversation-page.tsx (hunk 3 above)
Comment (abhinavrastogi-harness) [reply apps/gitness/src/pages/pull-request/sandbox-pull-request-conversation-page.tsx:82]: @cjlee01 @vardanbansal-harness let's meet sometime this week to move everything out of state for this page, and into a more suitable storage (context/reducer)

File: apps/gitness/src/layouts/PullRequestSandboxLayout.tsx (hunk 1 above)
Comment (sans-harness) [reply apps/gitness/src/layouts/PullRequestSandboxLayout.tsx:32]: Open to suggestions, can be done.

File: apps/gitness/src/pages/pull-request/sandbox-pull-request-conversation-page.tsx
Comment (sans-harness) [reply apps/gitness/src/pages/pull-request/sandbox-pull-request-conversation-page.tsx:1]: @cjlee01

Comment (cjlee01) [reply apps/gitness/src/pages/pull-request/sandbox-pull-request-conversation-page.tsx:1]: Will look into this, along with moving everything out of state as mentioned in below
cc . @sans-harness  @joetaylor-harness

File: apps/gitness/src/pages/pull-request/sandbox-pull-request-conversation-page.tsx (hunk 3 above)
Comment (cjlee01) [reply apps/gitness/src/pages/pull-request/sandbox-pull-request-conversation-page.tsx:82]: sure , can we sync tomoro @abhinavrastogi-harness  @vardanbansal-harness

**Pull request merged**